server/src/models/GameState.ts

```diff
@@ -127,8 +129,28 @@ export class GameState {
         delete this.heroes[id]
     }
 
+    addMinion(minion: Minion) {
+        this.minions[minion.id] = minion
```

Comment:
Just for a sanity check, if a minion already exists with this ID, remove it first